src/edge_containers_cli/cmds/cli.py

```diff
@@ -4,84 +4,112 @@
 import typer
 from natsort import natsorted
 
-import epics_containers_cli.globals as globals
-from epics_containers_cli.autocomplete import (
-    all_iocs,
+import edge_containers_cli.globals as globals
+from edge_containers_cli.autocomplete import (
+    all_svc,
     avail_IOCs,
     avail_versions,
     force_plain_completion,
-    running_iocs,
+    running_svc,
 )
-from epics_containers_cli.git import create_ioc_graph
-from epics_containers_cli.ioc.k8s_commands import IocK8sCommands
-from epics_containers_cli.ioc.local_commands import IocLocalCommands
-from epics_containers_cli.logging import log
-from epics_containers_cli.utils import cleanup_temp, drop_ioc_path
+from edge_containers_cli.cmds.k8s_commands import IocK8sCommands
+from edge_containers_cli.cmds.local_commands import IocLocalCommands
+from edge_containers_cli.git import create_svc_graph
+from edge_containers_cli.logging import log
+from edge_containers_cli.utils import cleanup_temp, drop_path
 
-ioc = typer.Typer()
+cli = typer.Typer(pretty_exceptions_show_locals=False)
 
 
-@ioc.command()
+@cli.command()
+def ps(
+    ctx: typer.Context,
+    all: bool = typer.Option(
+        False, "-a", "--all", help="list stopped IOCs as well as running IOCs"
```

Comment:
Still many reference exist to "Ioc", "IOC", "IOCs" in generic contexts

Reply:
Yep, that needs to go.

Reply:
I went with the term IOC/service in comments/command help or just dropped IOC from the sentence. I want to keep the word IOC to not lose context for long term EPICS engineers.